src/ghostfat.c

```diff
@@ -368,8 +433,8 @@ void uf2_read_block (uint32_t block_no, uint8_t *data)
     }
     else
     {
-      // generate the UF2 file data on-the-fly
-      sectionRelativeSector -= (NUM_FILES - 1) * BPB_SECTORS_PER_CLUSTER;
+      // CURRENT.UF2: generate data on-the-fly
+      sectionRelativeSector -= (UF2_FIRST_CLUSTER_NUMBER-2) * BPB_SECTORS_PER_CLUSTER;
```

Comment:
why not `info_cluster_start(NUM_FILES-1)`?
It would understand the sameness of these two code paths.

Reply:
Yeah, totally agreee